_episodes/02-filedir.md

```diff
@@ -61,7 +61,7 @@ which is Nelle's **home directory**:
 > ## Home Directory Variation
 >
 > The home directory path will look different on different operating systems.
-> On Linux it may look like `/home/nelle`,
+> On Linux and Mac it may look like `/home/nelle`,
```

Comment:
On a Mac, it's `/Users/nelle`. At least on the one I've used. This is shown in the diagram, so the original text is fine, pointing out the differences to the example used.